core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -388,7 +389,7 @@ object ConsoleConsumer extends Logging {
   private[tools] class ConsumerWrapper(topic: Option[String], partitionId: Option[Int], offset: Option[Long], whitelist: Option[String],
                                        consumer: Consumer[Array[Byte], Array[Byte]], val timeoutMs: Long = Long.MaxValue) {
     consumerInit()
-    var recordIter = consumer.poll(0).iterator
+    var recordIter = consumer.poll(Duration.ZERO).iterator
```

Comment:
Maybe we can just initialize this to an empty iterator? The call seems intended to join the group, but that will no longer happen when the call returns.

Reply:
Were you thinking about something like this?
```var recordIter = Collections.emptyList[ConsumerRecord[Array[Byte], Array[Byte]]]().iterator()```